options.c

```diff
@@ -155,14 +164,7 @@ void parse_options(int argc, char **argv)
 				_options.scalemode = s - scalemodes;
 				break;
 			case 'T':
-				if ((s = strrchr(optarg, ':')) != NULL) {
-					*s = '\0';
-					n = strtol(++s, &end, 0);
-					if (*end != '\0' || n < SUFFIX_EMPTY || n > SUFFIX_FULLPATH)
-						error(EXIT_FAILURE, 0, "Invalid argument for option -T suffixmode: %s", s);
-					_options.title_suffixmode = n;
-				}
-				_options.title_prefix = optarg;
+				title_deprecation_notice(); /* TODO: remove this option after sometimes */
```

Comment:
```suggestion
				title_deprecation_notice(); /* TODO(v30): remove this option */
```
or some later version.